dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/DataDeletionApi.kt

```diff
@@ -34,7 +34,17 @@ fun interface AdminDataManipulationApi {
         ],
     )
     @DeleteMapping()
-    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @PreAuthorize(
```

Comment:
Reminder für uns:  Fragen ob Company-Data-Uploader nur ihre eigenen datasets löschen können sollen

Reply:
Answered by Andreas: No the uploader can delete all datasets from the company

Reply:
so its fine the way it is